config/checkstyle-resources-suppressions.xml

```diff
@@ -804,6 +804,8 @@
     files="[\\/]it[\\/]resources[\\/]com[\\/]google[\\/]checkstyle[\\/]test[\\/]chapter2filebasic[\\/]rule231filetab[\\/]InputFileTabCharacter\.java"/>
   <suppress checks="FileLength"
     files="[\\/]it[\\/]resources[\\/]com[\\/]google[\\/]checkstyle[\\/]test[\\/]chapter2filebasic[\\/]rule232specialescape[\\/]InputIllegalTokenText\.java"/>
+  <suppress checks="FileLength"
+    files="[\\/]it[\\/]resources[\\/]com[\\/]google[\\/]checkstyle[\\/]test[\\/]chapter3filestructure[\\/]rule3421overloadsplit[\\/]InputOverloadsNeverSplit\.java"/>
```

Comment:
this will be a pain.

https://github.com/checkstyle/checkstyle/blob/104cf7723d2e7b38230e4792417462c52444043f/config/checkstyle-resources-checks.xml#L41-L44

we did this limit 120 for Inputs that are in test, that contributors are using to debug, so small files means easier to debug.
In ITs we do not need debug almost ever. So we do not need this limit.
let put single suppression for all ITs for FileLength.

Reply:
done 

https://github.com/checkstyle/checkstyle/blob/1922ffb7a499c0bb0993022ba97fe7728f66aa67/config/checkstyle-resources-suppressions.xml#L199-L202